python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -1115,6 +1129,43 @@ def evaluate(self, *, cache: MutableMapping[int, DataFrame]) -> DataFrame:
             return DataFrame.from_table(
                 plc.lists.explode_outer(df.table, index), df.column_names
             ).sorted_like(df, subset=subset)
+        elif self.name == "unpivot":
+            indices, pivotees, variable_name, value_name = self.options
+            npiv = len(pivotees)
+            df = self.df.evaluate(cache=cache)
+            if indices:
```

Comment:
Does indices default to None or to an empty sequence? If it's the latter, the `if` block would just work without needing a separate else IIUC. If it doesn't, that seems like an edge case bug to fix in plc. If it's the former, then nothing to do here.

Reply:
Empty sequence, so yes, I can simplify. _but_, does `cudf::reshape` work right with an empty table? I will check.

Reply:
Done.